maintainer/benchmarks/MC-acid-base-reservoir.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ionization degree alpha calculated from the Henderson-Hasselbalch equation for an ideal system
+def calc_ideal_alpha(pH, pKa):
+    return 1. / (1 + 10**(pKa - pH))
+
+def calc_donnan_coefficient(c_acid, I_res, charge = -1):
+    return charge*c_acid/(2*I_res) + np.sqrt( ( charge*c_acid/(2*I_res) )**2 + 1 )
```

Comment:
numpy hasn't been imported yet. It would be better to move these files after the imports and argparse logic